articles/quickstart/spa/angular-next/02-calling-an-api.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Call an API
+description: This tutorial demonstrates how to make API calls for protected resources on your server.
```

Comment:
the tutorial appears to show how to call our management API rather than on your server...